### DISCUSSION TITLE
Launching Minecraft with Pehkui and Connector

### DISCUSSION THREAD

**Discussion by MawarukoJonaga** [Sinytra/Connector#653] [Q&A] [answered]
Hello. When launching Minecraft with the Pehkui mod, I encounter an error. Do you have any recommendations on how to fix this?
![prtsc](https://github.com/Sinytra/Connector/assets/105217040/4a6a67cb-e542-41e0-965f-6dd94a705b9a)

**Comment by Su5eD** [comment, marked as answer]
Use the Forge version of Pehkui

**Reply by MawarukoJonaga** [reply]
Thank you
